discretisedfield/tools/tools.py

```diff
@@ -117,44 +117,28 @@ def topological_charge_density(field, /, method="continuous"):
 
     of = field.orientation  # unit field - orientation field
 
-    axis1 = field.mesh.region.dims[0]
-    axis2 = field.mesh.region.dims[1]
-    axis1_idx, axis2_idx = 0, 1
-
     if method == "continuous":
+        axis1 = field.mesh.region.dims[0]
+        axis2 = field.mesh.region.dims[1]
         return 1 / (4 * np.pi) * of.dot(of.diff(axis1).cross(of.diff(axis2)))
 
     elif method == "berg-luescher":
 
         q = df.Field(field.mesh, nvdim=1)
 
         # Area of a single triangle
-        area = 0.5 * field.mesh.cell[axis1_idx] * field.mesh.cell[axis2_idx]
+        area = 0.5 * field.mesh.cell[0] * field.mesh.cell[1]
+
+        n0, n1 = of.mesh.n[0], of.mesh.n[1]
```

Comment:
If you want to reduce the variables, you can directly use `of.mesh.n[0]` and `of.mesh.n[1]`below.